api_config.toml

```diff
@@ -1,5 +1,9 @@
 # This is the default toml config file that is used by the rest API server.
 #
+[restAPI]
+services = ["databrowser", "zarr-stream"]
+url = "http://localhost:7777"
```

Comment:
could you please separate the port from host here and define it as two params then we have more room for maneuver in conda build file?

Reply:
i proposed this, before u wanted to get the conda task. for the approach i wanted to use I needed it. Now I'm okey > resolved